src/extensions/cookie/bootstrap-table-cookie.js

```diff
@@ -212,12 +212,23 @@ const UtilsCookie = {
               element.value = cookie.text
               cachedFilters[cookie.field] = cookie.text
             } else if (cookie.text !== '' && element.tagName === 'SELECT') {
-              const option = document.createElement('option')
-              option.value = cookie.text
-              option.text = cookie.text
-              element.add(option, element[1])
-              element.selectedIndex = 1
-              cachedFilters[cookie.field] = cookie.text
+              var found = false
+              for (var i = 0; i < element.length; i++) {
+                var e = element[i]
+                if (e.value === cookie.text) {
+                  e.selected = true
+                  found = true
```

Comment:
Can replaced with return, so the folowing code will be skiped.

Reply:
yep. That's cleaner